rollup.config.mjs

```diff
@@ -10,6 +10,7 @@ export default {
   output: [
     {
       dir: 'lib',
+      entryFileNames: '[name].mjs',
```

Comment:
Using .mjs for now so bundlers interpret them as esm.

One day EDS will only have one build, and we can go back to .js then.